modules/core/src/v2/coins/eos.ts

```diff
@@ -8,6 +8,7 @@ import * as utxoLib from 'bitgo-utxo-lib';
 import * as url from 'url';
 import * as querystring from 'querystring';
 import * as _ from 'lodash';
+import * as ecc from'eosjs-ecc'
```

Comment:
wait sorry, missing whitespace here

Reply:
whoops not sure why the linter didnt see that